py/tests/plane_move.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+
+import pykms
+import random
+import time
+import sys
+import select
+import selectors
+
+if len(sys.argv) not in (1, 3):
+    print('Without parameters: list of available connectors')
+    print('{} [name of connector0] [name of connector1]\n'.format(sys.argv[0]))
+    sys.exit(-1)
```

Comment:
[ ] marks an optional parameter, < > marks a required parameter. The connector can also be given by index or number, not only by name, so just `"<connector1> <connector2>"` would be fine in my opinion.

I see a linefeed here, and in a few other prints. Are those correct? print() already adds a linefeed, doesn't it?

Reply:
Yes it does. It is leftover, removed the excess \n from the code